R/dataprep_table_formatting.R

```diff
@@ -74,6 +74,10 @@ fix_logical_update_existing <- function(sidora_table){
     sidora_table <- sidora_table %>% dplyr::mutate(dplyr::across(dplyr::contains(".Robot"), function(x) gsub(F, "No", x) %>% gsub(T, "Yes", .)))
   }
   
+  if (any(grepl("CT_scanned", colnames(sidora_table)))) {
+    sidora_table <- sidora_table %>% dplyr::mutate(dplyr::across(dplyr::contains(".CT_scanned"), function(x) gsub(F, "No", x) %>% gsub(T, "Yes", .)))
```

Comment:
Could we add some linebreaks here? Tall code is more easy to read than wide code.

Also: `contains` is from `tidyselect`. `dplyr` only reexports it. So maybe go back to the original package.

Reply:
This `gsub` solution is a bit wonky. Maybe sth like this would work:

```r
x <- c(TRUE, FALSE, NA)
dplyr::case_when(x ~ "yes", !x ~ "no")
```